RedHatInsights/rhsm-subscriptions
# SWATCH-749: Add sockets as a UOM for RHEL product ids

See https://issues.redhat.com/browse/SWATCH-749
---
# Testing

1. Before applying this patch, `gunzip` and import [swatch-749-test-data.sql.gz](https://github.com/RedHatInsights/rhsm-subscriptions/files/10501352/swatch-749-test-data.sql.gz) .  Let me know if you have problems with the import.  I believe this should work but it's my first time trying an export with a new method.
2. Run
    ```shell
    http :8000/api/rhsm-subscriptions/v1/instances/products/RHEL%20for%20IBM%20z \
    beginning==2022-01-01T00:00:00.000Z ending==2023-01-10T23:59:59.999Z dir==desc sort==last_seen uom==sockets \
    x-rh-identity:$(echo -n '{"identity":{"account_number":"account123","type":"User","user":{"is_org_admin":true},"internal":{"org_id":"org123"}}}' | base64 -w0)
    ```
3. Observe that no measurements are present on any of the returned elements.
4. Apply this patch.
5. Rerun the `http` command and observe that the measurements are present.
